sdks/python/apache_beam/io/iobase.py

```diff
@@ -1289,9 +1289,9 @@ class RestrictionProgress(object):
   """
   def __init__(self, **kwargs):
     # Only accept keyword arguments.
-    self._fraction = kwargs.pop('fraction', None)
-    self._completed = kwargs.pop('completed', None)
-    self._remaining = kwargs.pop('remaining', None)
+    self._fraction = kwargs.pop('fraction', None)  # type: Optional[float]
+    self._completed = kwargs.pop('completed', None)  # type: Optional[int]
```

Comment:
I don't know that we want to restrict this to integral values.

Reply:
I was a pretty confused by this class, and I think there may actually be some bugs in its handling of `None`-values, so I tried to do the bare minimum of what I thought was right.  No worries. I will revert this and leave this as a final typing task to someone who understands it.